static/js/util/util.js

```diff
@@ -249,6 +249,6 @@ export function calculateDegreeInclusions(profile: Profile) {
 /**
  * Calls an array of functions in series with a given argument and returns an array of the results
  */
-export function callFunctionArray(functionArray, arg) {
+export function callFunctionArray<T,R>(functionArray: Array<(t: T) => R>, arg: any): R[] {
```

Comment:
Each function needs to share a common argument type to work with `arg`. I guess they don't need to have the same return type necessarily but they do for the functions we're currently using. What do you think?